### ISSUE TITLE
Should the Lego WeDo "set motor power" block not turn on the motor?

### ISSUE THREAD

**Issue by ericrosenbaum** [LLK/scratch-vm#708]
Currently, the Lego WeDo extension block "set motor power" sets the motor power and also turns the motor on. The WeDo extension in Scratch 2.0 also had this behavior. The motor turning on can seem unexpected, if you already have the mental model of "motor power" being an internal setting that applies to future motor commands. If you do not have this mental model, you might expect setting the motor power to take effect immediately (kind of like pressing the high button or the low button on a desk fan)- this may have been the original justification for the behavior. 

Should we keep this behavior? Or should the "set motor power" block only set the motor power?

**Comment by ntlrsk** [comment]
Yeah, this is one where either choice turns out not to be what many people expect. We have tried just having it set the state (which I feel is more logical), but often when we've tested that with kids and educators, I've seen them click the block and ask "Why didn't it work?" So that's why we implemented it this way. 

Maybe for example setting the brightness of a light -- or setting a heater to maximum power they think of seeing the effects right away. 

(I can see for some Scratch blocks "set" will show a difference right away - while for others it requires another block before you see/hear that state. Logically consistent but maybe not as intuitively so.)
